### PR TITLE
Add Education Form Component

### PR DESCRIPTION
**Pull request by cubeydice**
Closes Issue #10 

**Description:**
- Adds an EducationForm component
- Form allows users to add education, which includes course, school, start date, end date, grade, and logo URL.
- Submits a POST request to the /resume/education endpoint

**Changes Made:**
- Created EducationForm.jsx
- Implements a form for adding education detail
- Sends a POST request with the form data to the server
- Updated App.js to include new form upon toggling a button

**Future Updates:**
- Error handling with frontend error message showing per input; implement useEffect to display
- Use redux to manage application state
- Create modal for submission success message
- Display education submissions on App instead of "Education Placeholder"

**Considerations:**
- Use the CSS styling from the experience pull request for consistency across all forms

image

### CODE REVIEW

**Comment by cubeydice** [comment]
@Pradyuman7 This pull request is ready for your review

File: src/components/education/EducationForm.jsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        fetch('resume/education', {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify(formData)
+        })
+        .then(response => response.json())
+        .then(data => {
+            console.log('Success:', data)
```
Comment (Pradyuman7) [inline src/components/education/EducationForm.jsx]: We could show an alertbox here since this success is relevant for the user

File: src/App.css [hunk 2]
```diff
@@ -27,3 +27,37 @@ button {
   border-radius: 4px;
   background-color: rgb(248 185 42);
 }
+
```
Comment (Pradyuman7) [inline src/App.css:30]: just a general nit: usually using rem is better than using px. A good [example](https://dev.to/ochukodotspace/the-battle-of-the-units-px-vs-rem-vs-em-3ka8#:~:text=Now%20PX%20and%20REM%20basically,accessibility%20comes%20into%20the%20picture)

File: src/components/education/EducationForm.jsx (hunk 1 above)
Comment (cubeydice) [reply src/components/education/EducationForm.jsx]: Added alert box for successful and unsuccessful submission

**Comment by cubeydice** [comment]
@Pradyuman7 PR updated for your review!

**Review by Pradyuman7 — APPROVED** [review]
LGTM 💯 

can you please resolve the conflicts, then I'll merge the PR

**Comment by cubeydice** [comment]
@Pradyuman7 Conflicts are resolved!

**Pull request merged**